Importer/EventListener/InterpreterExceptionSubscriber.php

```diff
@@ -45,15 +44,16 @@ public function handleException(InterpreterExceptionEvent $event)
     private function resolveInterpreterExceptionMessage(InterpreterException $exception)
     {
         if ($exception instanceof DateTimeFormatException) {
-            return "Error matching date \"{$exception->getValue()}\" to format \"{$exception->getFormat()}\".";
+            return "Error matching date “{$exception->getValue()}” to format “{$exception->getFormat()}”.";
         } elseif ($exception instanceof LookupFieldException) {
             return $this->buildLookupFieldExceptionMessage($exception);
         } elseif ($exception instanceof RowSizeMismatchException) {
             return "Row is expected to contain {$exception->getExpectedSize()}, found to have {$exception->getFoundSize()} fields instead.";
         } elseif ($exception instanceof UnknownColumnException) {
+            // TODO: This should not be data error. Its more like configuration error. Check if this can be reproduced with the current implementation.
```

Comment:
No. On this case we throw an `UnknownOrInaccessibleFieldException`. You can check:

- vendor/netdudes/importer-bundle/Netdudes/ImporterBundle/Importer/Interpreter/EntityDataInterpreter.php:307